src/components/structures/ContextMenu.tsx

```diff
@@ -25,7 +25,7 @@ import { Key } from "../../Keyboard";
 import { Writeable } from "../../@types/common";
```

Comment:
Testing strategy:

 - ✅ `UserMenu` ([make sure can still navigate with custom user status messages](https://github.com/vector-im/element-web/issues/19881))
 - ✅ `MessageActionBar`
 - ✅ `ReactionsRow`
    - The reaction picker does a lot of it's own thing but everything seems in order to how it worked before
 - ✅ `RoomTile` -> `renderGeneralMenu`
 - ✅ `RoomTile` -> `renderNotificationsMenu`
 - ✅ `RoomSubList` (the settings `...` on each room list heading for sorting stuff)
 - ✅ `RoomListHeader` (the `+` next to the "Rooms" heading)
 - ✅ `ThreadPanelHeader` "My threads" vs "All threads" (accessed via the top bar, not a thread in the timeline)
 - ✅ `NetworkDropdown` in the explore rooms modal
 - ✅ Spaces `QuickSettings` (under feature flag, at bottom of Spaces panel)
 - ✅ `SpaceCreateMenu`
 - ✅ `WidgetContextMenu` (`/addwidget https://something`, open the room info right panel, under the "Widgets" section, hover your widget, open the `...` menu)
    - Since it only displays on hover, can't get to it via keyboard but the `ContextMenu` options works fine once it's open
 - ✅ `CallContextMenu` (start a 1:1 call (has to be answered) to see the `...` button labelled "More")
 - ✖ `DialpadContextMenu` (edit [`src/components/views/voip/CallView/CallViewButtons.tsx#L216`](https://github.com/matrix-org/matrix-react-sdk/blob/7b94e13a8438f98dfd6b7543d011fc8835968d38/src/components/views/voip/CallView/CallViewButtons.tsx#L216) for it to always show, then make a call and look for the dialpad icon button)
    - Was not keyboard accessible before
 - ✖ `Stickerpicker`
    - Was not keyboard accessible before

Reply:
The Stickerpicker isn't owned by Element-Web either, its an iframe